mkdocs.yml

```diff
@@ -407,6 +407,8 @@ nav:
       - documentation/functions/aria-suite/aria-operations-logs/Update-vRLILogForwarder.md
     - Aria Operations for Networks:
       - documentation/functions/aria-suite/aria-operations-networks/Add-AriaNetworksVcenterDataSource.md
+      - documentation/functions/aria-suite/aria-operations-networks/Add-AriaNetworksLdapConfiguration.md
+      - documentation/functions/aria-suite/aria-operations-networks/Undo-AriaNetworksLdapConfiguration.md
```

Comment:
Could you move this down so it's alphabetical?

Reply:
resolved in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/6c0cd2aac71935652b35a46ac8a5fef1c435e3db